js/main.js

```diff
@@ -443,6 +454,7 @@ function updateBodyClasses() {
 }
 
 function doTask(task) {
+    if (task == null || task.done) return;

```

Comment:
```suggestion
    if (task === null || task.isDone()) return;

```

--> entsprechend im LayeredTask noch eine methode einführen und LayeredTask.done als private behandeln.